### ISSUE TITLE
Improve Live tab

### ISSUE THREAD

**Issue by loomchild** [loomchild/sojourner-web#133]
Show future events, not almost ending.

**Comment by digitalica** [comment]
i agree. it showed too much running events (event almost finished ones) and not enought into the future. 

maybe dont show events ending in the next 15 mins, or over 50% of their time or something? And then a bit more in the furure. That way it would be more usefull to use the live tab to decide where to go after this (even if after this is an hour away)

**Comment by loomchild** [comment]
Yes, this tab needs some love. Thanks for your suggetions. I'll look into this before the next edition.

**Comment by loomchild** [comment]
I tweaked it a bit:
- show events starting within 1h
- hide events ending in less than 15 minutes (unless they haven't started yet or are favorite)

I'll close it for now, but please share your suggestions.
